ChopChop/ViewModels/Recipes/RecipeFormViewModel.swift

```diff
@@ -162,6 +168,52 @@ class RecipeFormViewModel: ObservableObject {
         }
     }
 
+    private func validateRecipe() -> Bool {
+        var hasErrors = false
```

Comment:
Is there a reason for this variable? It seems like 
`hasErrors == !formErrors.isEmpty`

Reply:
Ah you're right.